src/charm.py

```diff
@@ -1389,27 +1377,39 @@ def _plugin_reconciliation(self) -> None:
             self._plugin_akismet_reconciliation()
             self._plugin_openid_reconciliation()
 
+    def _storage_mounted(self) -> bool:
+        """Check if the upload storage mounted in the wordpress container.
+
+        Returns:
+            True if the storage "upload" is attached to the container.
+        """
+        container = self._container()
+        if not container.can_connect():
+            return False
+        mount_info: str = container.pull("/proc/mounts").read()
+        return "/var/www/html/wp-content/uploads" in mount_info
```

Comment:
Would it be better to parse the content of `/proc/mounts` so we can check for a exact match?
Since `/proc/mounts` should have a well defined format, and this way of checking can have false positive in theory.

Reply:
The problem of parsing the `/proc/mounts` is that it can be quite complex, as file names in Linux can contain whitespaces and newlines.